packages/terra-collapse/tests/nightwatch/collapse-spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
```

Comment:
I view these test primarily for generating screenshots / test pages for manual functional verification. In regards to testing particular props or attributes the content or button should have, I think that should be covered with Jest tests. I'm open to ideas on the role of the nightwatch tests.